example/src/screens/timelineCalendar.js

```diff
@@ -0,0 +1,312 @@
+    }
+
+    return (
+      <TouchableOpacity
+        onPress={() => this.itemPressed(item.title)}
+        style={styles.item}>
+        <View>
+          <Text style={styles.itemHourText}>{item.hour}</Text>
+          <Text style={styles.itemDurationText}>{item.duration}</Text>
+        </View>
+        <Text style={styles.itemTitleText}>{item.title}</Text>
+        <View style={styles.itemButtonContainer}>
+          <Button title={'Info'} onPress={this.buttonPressed} />
+        </View>
+      </TouchableOpacity>
+    );
+  };
+
+  // getMarkedDates = () => {
+  //   const marked = {};
+  //   ITEMS.forEach(item => {
+  //     // only mark dates with data
+  //     if (item.data && item.data.length > 0 && !_.isEmpty(item.data[0])) {
+  //       marked[item.title] = {marked: true};
+  //     }
+  //   });
+  //   return marked;
+  // };
+
+  getTheme = () => {
+    const themeColor = '#0059ff';
+    const lightThemeColor = '#e6efff';
+    const disabledColor = '#a6acb1';
+    const black = '#20303c';
+    const white = '#ffffff';
+
+    return {
+      // arrows
+      arrowColor: black,
+      arrowStyle: {padding: 0},
+      // month
+      monthTextColor: black,
+      textMonthFontSize: 16,
+      textMonthFontFamily: 'HelveticaNeue',
+      textMonthFontWeight: 'bold',
+      // day names
+      textSectionTitleColor: black,
+      textDayHeaderFontSize: 12,
+      textDayHeaderFontFamily: 'HelveticaNeue',
+      textDayHeaderFontWeight: 'normal',
+      // today
+      todayBackgroundColor: lightThemeColor,
+      todayTextColor: themeColor,
+      // dates
+      dayTextColor: themeColor,
+      textDayFontSize: 18,
+      textDayFontFamily: 'HelveticaNeue',
+      textDayFontWeight: '500',
+      textDayStyle: {marginTop: Platform.OS === 'android' ? 2 : 4},
+      // selected date
+      selectedDayBackgroundColor: themeColor,
+      selectedDayTextColor: white,
+      // disabled date
+      textDisabledColor: disabledColor,
+      // dot (marked date)
+      dotColor: themeColor,
+      selectedDotColor: white,
+      disabledDotColor: disabledColor,
+      dotStyle: {marginTop: -2},
+    };
+  };
+
+  render() {
+    return (
+      <CalendarProvider
+        // date={ITEMS[0].title}
+        onDateChanged={this.onDateChanged}
+        onMonthChange={this.onMonthChange}
+        theme={{todayButtonTextColor: '#0059ff'}}
+        showTodayButton
+        disabledOpacity={0.6}
+        // todayBottomMargin={16}
+      >
+        <ExpandableCalendar
+          // horizontal={false}
+          // hideArrows
+          // disablePan
+          // hideKnob
+          // initialPosition={ExpandableCalendar.positions.OPEN}
+          firstDay={1}
+          // markedDates={this.getMarkedDates()} // {'2019-06-01': {marked: true}, '2019-06-02': {marked: true}, '2019-06-03': {marked: true}};
+          markedDates={() => {}} // {'2019-06-01': {marked: true}, '2019-06-02': {marked: true}, '2019-06-03': {marked: true}};
```

Comment:
Why not mark the dates that have events?

Reply:
@Inbal-Tish Right now, Event's start and end time have both date and time, and there is not function to extract all dates between Event's start and end time.